qbittorrent.json

```
@@ -0,0 +1,64 @@
+{
+    "qBittorrent": {
+        "containers": {
+            "qbittorrent-linuxserver.io": {
+                "image": "linuxserver/qbittorrent",
+                "launch_order": 1,
+                "opts": [
+                    [
+                        "-e",
+                        "UMASK_SET=007"
+                    ],
+                    [
+                        "-e",
+                        "WEBUI_PORT=8090"
```

Set WEBUI_Port to an exotic number matching the published `"ui"` port below.
```suggestion
                        "WEBUI_PORT=12500"
```